diff --git a/fitPlugins/exp_saturation/exp_saturation.pro b/fitPlugins/exp_saturation/exp_saturation.pro
index ac0d1446..5b0e181f 100644
--- a/fitPlugins/exp_saturation/exp_saturation.pro
+++ b/fitPlugins/exp_saturation/exp_saturation.pro
@@ -10,7 +10,7 @@ CONFIG += release
 DESTDIR = ../
 
 INSTALLS += target
-unix:  target.path = /usr/lib$${libsuff}/AlphaPlot/plugins
+unix:  target.path = /app/lib$${libsuff}/AlphaPlot/plugins
 win32: target.path = ../../output/plugins
 
 win32:INCLUDEPATH += ../../3rdparty/gsl/include
diff --git a/fitPlugins/explin/explin.pro b/fitPlugins/explin/explin.pro
index 176d5a55..3f6db8fe 100644
--- a/fitPlugins/explin/explin.pro
+++ b/fitPlugins/explin/explin.pro
@@ -10,7 +10,7 @@ CONFIG += release
 DESTDIR = ../
 
 INSTALLS += target
-unix:  target.path = /usr/lib$${libsuff}/AlphaPlot/plugins
+unix:  target.path = /app/lib$${libsuff}/AlphaPlot/plugins
 win32: target.path = ../../output/plugins
 
 win32:INCLUDEPATH += ../../3rdparty/gsl/include
diff --git a/fitPlugins/fitRational0/fitRational0.pro b/fitPlugins/fitRational0/fitRational0.pro
index b0adbffd..d9b197a7 100644
--- a/fitPlugins/fitRational0/fitRational0.pro
+++ b/fitPlugins/fitRational0/fitRational0.pro
@@ -13,7 +13,7 @@ CONFIG           += dll
 DESTDIR           = ../
 
 INSTALLS += target
-unix:target.path=/usr/lib$${libsuff}/AlphaPlot/plugins
+unix:target.path=/app/lib$${libsuff}/AlphaPlot/plugins
 win32: target.path = ../../output/plugins
 
 win32:INCLUDEPATH += ../../3rdparty/gsl/include
diff --git a/fitPlugins/fitRational1/fitRational1.pro b/fitPlugins/fitRational1/fitRational1.pro
index 190c2eec..b13eb6b2 100644
--- a/fitPlugins/fitRational1/fitRational1.pro
+++ b/fitPlugins/fitRational1/fitRational1.pro
@@ -14,7 +14,7 @@ DESTDIR           = ../
  
 INSTALLS += target
 # where to install the plugins
-unix:target.path=/usr/lib$${libsuff}/AlphaPlot/plugins
+unix:target.path=/app/lib$${libsuff}/AlphaPlot/plugins
 win32: target.path = ../../output/plugins
 
 win32:INCLUDEPATH += ../../3rdparty/gsl/include
diff --git a/fitPlugins/planck_wavelength/planck_wavelength.pro b/fitPlugins/planck_wavelength/planck_wavelength.pro
index d4e59b7e..93e29216 100644
--- a/fitPlugins/planck_wavelength/planck_wavelength.pro
+++ b/fitPlugins/planck_wavelength/planck_wavelength.pro
@@ -10,7 +10,7 @@ CONFIG += release
 DESTDIR = ../
 
 INSTALLS += target
-unix:  target.path = /usr/lib$${libsuff}/AlphaPlot/plugins
+unix:  target.path = /app/lib$${libsuff}/AlphaPlot/plugins
 win32: target.path = ../../output/plugins
 
 win32:INCLUDEPATH += ../../3rdparty/gsl/include
